fake-koji/src/main/java/org/fakekoji/jobmanager/project/JDKProjectParser.java

```diff
@@ -112,8 +112,7 @@ public JDKProjectParser(
     private Consumer<PlatformConfig> getPlatformsConsumer() {
         return ManagementUtils.managementConsumerWrapper(
                 (PlatformConfig platformConfig) -> {
-                    jobBuilder.setPlatformProvider(platformConfig.getProvider());
-                    jobBuilder.setPlatform(platformConfig.getId());
+                    jobBuilder.setPlatform(platformConfig.getId(), platformConfig.getProvider());
```

Comment:
From the scope of code in this patch, I'm nto sure I got the change:                     -jobBuilder.setPlatformProvider(platformConfig.getProvider());
-jobBuilder.setPlatform(platformConfig.getId());

replaced by 

(platformConfig.getId(), platformConfig.getProvider()) Where I can not see hadnling og this tuple